client/client.go

```diff
@@ -0,0 +1,106 @@
+package client
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"strings"
+	"time"
+
+	websocket "github.com/gorilla/websocket"
+
+	api "github.com/derekparker/delve/api"
```

Comment:
No need to directly specify package name here, imported package is already named "api".